a/apache-solr/solr_ubi_8.5.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package          : solr
+# Version          : releases/solr/9.2.0
+# Source repo      : https://github.com/apache/solr
+# Tested on        : UBI 8.5
+# Language         : Java
+# Travis-Check     : False 
+# Script License   : Apache License, Version 2 or later
+# Maintainer       : Shubham Garud <Shubham.Garud@ibm.com>
+#
+# Disclaimer       : This script has been tested in root mode on given
+# ==========         platform using the mentioned version of the package.
+#                    It may not work as expected with newer versions of the
+#                    package and/or distribution. In such case, please
+#                    contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=solr
+PACKAGE_VERSION=${1:-releases/solr/9.2.0}
+PACKAGE_URL=https://github.com/apache/solr
+
+# Install dependencies
+yum install -y wget git java-11-openjdk java-11-openjdk-devel
+
+echo `java -XshowSettings:properties -version 2>&1 > /dev/null | grep 'java.home'`
+echo `PATH`
+export JAVA_HOME=/usr/lib/jvm/java-11-openjdk-11.0.19.0.7-1.el8_7.ppc64le
+export PATH=/usr/lib/jvm/java-11-openjdk-11.0.12.0.7-0.el8_4.ppc64le/bin/:$PATH
+
+
+git clone $PACKAGE_URL
+cd  $PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+
+chmod +x gradlew
+
+if ! ./gradlew localSettings; then
```

Comment:
@shubham-garud Is this command mentioned readme or CI workflow? Can you please point the reference

Reply:
Yes @nikhil-kalbande 
Here is the reference :
https://github.com/apache/solr/actions/runs/4028015923/workflow